src/main/java/ch/hsr/epj/localshare/demo/gui/application/MainWindowController.java

```diff
@@ -122,6 +125,12 @@ public void initialize(URL location, ResourceBundle resources) {
   }
 
   private void startHttpServer() {
-    this.httpServerController = new HttpServerController();
+    httpServerController = new HttpServerController();
+
+    // FIXME: hack to get a file to share without a peer
+    List<File> files = new ArrayList<>();
+    File file = new File("C:\\Users\\marco\\rocket.jpg");
```

Comment:
Please don't use static file paths in code. Other users do not have this image nor the same path.